snapshots/collections-schema.yml

```diff
@@ -1273,89 +1325,89 @@ fields:
+      collection: gp_credits_additions
       conditions: null
       display: null
       display_options: null
-      field: id
+      field: comment
       group: null
-      hidden: true
+      hidden: false
       interface: input
-      note: null
+      note: Why credits are added?
       options: null
-      readonly: true
-      required: false
-      sort: 1
+      readonly: false
+      required: true
+      sort: 10
       special: null
       translations: null
       validation: null
       validation_message: null
       width: full
     schema:
-      name: id
-      table: gp_credits
-      data_type: int unsigned
+      name: comment
+      table: gp_credits_additions
+      data_type: varchar
       default_value: null
-      max_length: null
-      numeric_precision: 10
-      numeric_scale: 0
+      max_length: 255
+      numeric_precision: null
+      numeric_scale: null
       is_nullable: false
       is_unique: false
-      is_primary_key: true
+      is_primary_key: false
       is_generated: false
       generation_expression: null
-      has_auto_increment: true
+      has_auto_increment: false
       foreign_key_table: null
       foreign_key_column: null
-  - collection: gp_credits
-    field: user_updated
-    type: string
+  - collection: gp_credits_additions
+    field: consumed
+    type: boolean
     meta:
-      collection: gp_credits
+      collection: gp_credits_additions
       conditions: null
-      display: user
+      display: null
       display_options: null
-      field: user_updated
+      field: consumed
       group: null
-      hidden: true
-      interface: select-dropdown-m2o
+      hidden: false
+      interface: boolean
       note: null
       options:
-        template: '{{avatar.$thumbnail}} {{first_name}} {{last_name}}'
-      readonly: true
+        label: Consumed
+      readonly: false
       required: false
-      sort: 4
+      sort: 11
       special:
-        - user-updated
+        - cast-boolean
       translations: null
       validation: null
       validation_message: null
-      width: half
+      width: full
     schema:
-      name: user_updated
-      table: gp_credits
-      data_type: char
-      default_value: null
-      max_length: 36
-      numeric_precision: null
-      numeric_scale: null
-      is_nullable: true
+      name: consumed
+      table: gp_credits_additions
+      data_type: boolean
+      default_value: true
+      max_length: null
+      numeric_precision: 3
+      numeric_scale: 0
+      is_nullable: false
       is_unique: false
       is_primary_key: false
       is_generated: false
       generation_expression: null
       has_auto_increment: false
-      foreign_key_table: directus_users
-      foreign_key_column: id
-  - collection: gp_tokens
+      foreign_key_table: null
+      foreign_key_column: null
+  - collection: gp_credits_additions
     field: date_created
     type: timestamp
     meta:
-      collection: gp_tokens
```

Comment:
Missed this in the previous PR - any good reason why we store token hashes as base64 strings instead of native binary column? If not, let's change it to `varbinary(32)`

Reply:
The only available type from perspective of Directus is `binary`. But it maps internally to `blob` instead of `varbinary`.

Reply:
Ok I guess keep it.